src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritProjectListUpdater.java

```diff
@@ -130,7 +138,7 @@ public void run() {
 
             try {
                 synchronized (this) {
-                    wait(UPDATE_DELAY);
+                    wait(TimeUnit.SECONDS.toMillis(getConfig().getProjectListRefreshInterval()));
```

Comment:
This value could be changed during runtime to ```0``` which should abort it, not cause the thread to never sleep.

How should a change from ```0``` to ```>0``` be handled? I guess admins are used to clicking that reconnect button?